ARM64 support/package creation

### Use-cases
Please support ARM64 so we could use this tool in extension "HashiCorp Terraform" for Visual Studio Code on Raspberry Pi with 64 bits OS (Ubuntu or Raspberry OS)

### Attempted Solutions
I tried to build the tool on Ubuntu 20.04.1 arm64, on Raspberry pi 4 / 8 GB.
I run make command after cloning git repository.

```
2020/12/27 22:39:24 root_module.go:613: parsing file "main.tf"
--- PASS: TestSchemaForPath_uninitialized (5.90s)
PASS
coverage: 57.0% of statements
ok      github.com/hashicorp/terraform-ls/internal/terraform/rootmodule 6.870s  coverage: 57.0% of statements
=== RUN   TestURIFromPath
--- PASS: TestURIFromPath (0.00s)
PASS
coverage: 100.0% of statements
ok      github.com/hashicorp/terraform-ls/internal/uri  0.032s  coverage: 100.0% of statements
?       github.com/hashicorp/terraform-ls/internal/watcher    [no test files]
FAIL
make: *** [Makefile:7: test] Error 1
```

I decided to execute "go build" command, and the binary terraform-ls was created correctly.
I'm going to test it later.

--- a/.goreleaser.yml
+++ b/.goreleaser.yml
@@ -14,10 +14,15 @@ builds:
       - -trimpath
     ldflags:
       - '-s -w -X "main.version={{ .RawVersion }}" -X "main.prerelease={{ if .IsSnapshot }}snapshot.{{ .ShortCommit }}{{ else }}{{ .Prerelease }}{{ end }}"'
-    targets:
-      - darwin_amd64
-      - windows_386
-      - windows_amd64
+    goarch:
+      - '386'
+      - amd64
+    goos:
+      - darwin
+      - windows
+    ignore:
+      - goarch: '386'
+        goos: darwin
     hooks:
       post: |
         docker run
@@ -33,16 +38,20 @@ builds:
       - -trimpath
     ldflags:
       - '-s -w -X "main.version={{ .RawVersion }}" -X "main.prerelease={{ if .IsSnapshot }}snapshot.{{ .ShortCommit }}{{ else }}{{ .Prerelease }}{{ end }}"'
-    targets:
-      - freebsd_386
-      - freebsd_amd64
-      - freebsd_arm
-      - linux_386
-      - linux_amd64
-      - linux_arm
-      - openbsd_386
-      - openbsd_amd64
-      - solaris_amd64
+    goarch:
+      - '386'
+      - amd64
+      - arm
+    goos:
+      - freebsd
+      - openbsd
+      - linux
+      - solaris
+    ignore:
+      - goarch: '386'
+        goos: darwin
+      - goarch: arm
+        goos: openbsd
 
 archives:
   -

--- a/.goreleaser.yml
+++ b/.goreleaser.yml
@@ -42,6 +42,7 @@ builds:
       - '386'
       - amd64
       - arm
+      - arm64
     goos:
       - freebsd
       - openbsd
@@ -52,6 +53,10 @@ builds:
         goos: darwin
       - goarch: arm
         goos: openbsd
+      - goarch: arm64
+        goos: openbsd
+      - goarch: arm64
+        goos: freebsd
 
 archives:
   -

